src/docs/getting-started.md

```diff
@@ -134,3 +134,14 @@ You will see the URL `http://localhost:4200`, if you open this in the browser yo
 Browser.
 
 Congratulations 👏, you've just written your first Broccoli pipeline!
+
+For a demo, checkout this sanbox from [codesandbox.io](https://codesandbox.io)
+
+<div class="mobile-show">
+<a href="https://codesandbox.io/s/github/broccolijs/broccoli-tutorial/tree/master/">
```

Comment:
this makes me pretty motivated to add livereload :P

Reply:
Yeah, live reload should be part of the core.
Can we use https://github.com/broccolijs/broccoli/issues/352 to setup a broccoli 2.0 roadmap? Or should we make a new issue?

Reply:
One approach would be to re-visit as part of (or after) unifying the watcher effort.

Reply:
But if you see a fast-path, and are motivated. Please don't let me block you.

Reply:
I'm not sure what you're asking me to do tbh, is there an issue with using this now?

Reply:
sorry, i misread:

> is there an issue with using this now?

No issue